app-mgmt doesn't build

```
error[E0609]: no field `ksz8463` on type `&gimletlet_mgmt::Bsp`
   --> task\net\src\bsp\gimletlet_mgmt.rs:182:35
    |
182 |         ringbuf_entry!(match self.ksz8463.read_dynamic_mac_table(0) {
    |                                   ^^^^^^^ unknown field
    |
    = note: available fields are: `mgmt`, `leds`
help: one of the expressions' fields has a field of the same name
    |
182 |         ringbuf_entry!(match self.mgmt.ksz8463.read_dynamic_mac_table(0) {
    |                                   +++++

error[E0599]: no variant or associated item named `Ksz8463EmptyMacTable` found for enum `gimletlet_mgmt::Trace` in the current scope
   --> task\net\src\bsp\gimletlet_mgmt.rs:184:32
    |
22  | enum Trace {
    | ---------- variant or associated item `Ksz8463EmptyMacTable` not found here
...
184 |             Ok(None) => Trace::Ksz8463EmptyMacTable,
    |                                ^^^^^^^^^^^^^^^^^^^^
    |                                |
    |                                variant or associated item not found in `gimletlet_mgmt::Trace`
    |                                help: there is a variant with a similar name: `Ksz8463MacTable`

error[E0616]: field `0` of struct `MAC_SERDES_PCS_STATUS` is private
   --> task\net\src\bsp\gimletlet_mgmt.rs:229:41
    |
229 |                     any_link |= (status.0 & (1 << 2)) != 0;
    |                                         ^ private field

error[E0616]: field `0` of struct `MAC_SERDES_PCS_STATUS` is private
   --> task\net\src\bsp\gimletlet_mgmt.rs:230:42
    |
230 |                     any_comma |= (status.0 & (1 << 0)) != 0;
    |                                          ^ private field

Some errors have detailed explanations: E0599, E0609, E0616.
For more information about an error, try `rustc --explain E0599`.
error: could not compile `task-net` due to 4 previous errors
Error: failed to build net

Caused by:
    command failed, see output for details
error: process didn't exit successfully: `target\debug\xtask.exe dist app\gimletlet\app-mgmt.toml` (exit code: 1)
```

--- a/task/net/src/bsp/gimletlet_mgmt.rs
+++ b/task/net/src/bsp/gimletlet_mgmt.rs
@@ -39,6 +39,7 @@ enum Trace {
         port: u8,
         counter: MIBCounterValue,
     },
+    Ksz8463EmptyMacTable,
     Ksz8463MacTable(ksz8463::MacTableEntry),
 
     Vsc8552Status {
@@ -180,7 +181,7 @@ impl Bsp {
         }
 
         // Read the MAC table for fun
-        ringbuf_entry!(match self.ksz8463.read_dynamic_mac_table(0) {
+        ringbuf_entry!(match self.mgmt.ksz8463.read_dynamic_mac_table(0) {
             Ok(Some(mac)) => Trace::Ksz8463MacTable(mac),
             Ok(None) => Trace::Ksz8463EmptyMacTable,
             Err(err) => Trace::KszErr { err },
@@ -190,7 +191,7 @@ impl Bsp {
         let mut any_link = false;
         let rw = &mut MiimBridge::new(eth);
         for i in [0, 1] {
-            let mut phy = self.mgmt.vsc85x2.phy(i, rw).phy;
+            let phy = self.mgmt.vsc85x2.phy(i, rw).phy;
             let port = phy.port;
 
             ringbuf_entry!(match phy.read(phy::STANDARD::MODE_STATUS()) {
@@ -227,8 +228,8 @@ impl Bsp {
                 .read(phy::EXTENDED_3::MAC_SERDES_PCS_STATUS())
             {
                 Ok(status) => {
-                    any_link |= (status.0 & (1 << 2)) != 0;
-                    any_comma |= (status.0 & (1 << 0)) != 0;
+                    any_link |= status.mac_link_status() != 0;
+                    any_comma |= status.mac_pcs_sig_detect() != 0;
                     Trace::Vsc8552MacPcsStatus { port, status }
                 }
                 Err(err) => Trace::Vsc8552Err { err },
@@ -265,8 +266,8 @@ impl Bsp {
         &mut self,
         port: u8,
         reg: PhyRegisterAddress<u16>,
-        eth: &Ethernet,
-    ) -> Result<T, PhyError> {
+        eth: &crate::eth::Ethernet,
+    ) -> Result<u16, PhyError> {
         self.mgmt.phy_read(port, reg, eth)
     }
 
@@ -275,8 +276,8 @@ impl Bsp {
         port: u8,
         reg: PhyRegisterAddress<u16>,
         value: u16,
-        eth: &Ethernet,
-    ) -> Result<T, PhyError> {
-        self.mgmt.phy_write(port, reg, eth, value)
+        eth: &crate::eth::Ethernet,
+    ) -> Result<(), PhyError> {
+        self.mgmt.phy_write(port, reg, value, eth)
     }
 }

